policies/coding-style.md

```diff
@@ -586,10 +586,8 @@ it portable enough to meet our multi-platform support goals.
 
 ## Chapter 14:  Portability
 
-To maximise portability the version of C defined in ISO/IEC 9899:1990
-should be used. This is more commonly referred to as C90. ISO/IEC 9899:1999
-(also known as C99) is not supported on some platforms that OpenSSL is
-used on and therefore should be avoided.
+The version of C defined in ISO/IEC 9899:1999 (C99) should be used.
+Features that became optional in ISO/IEC 9899:2011 (C11) should not be used.
```

Comment:
Should this change apply to 3.1 and newer releases only?

Reply:
That was at least the intention. How do we handle such changes in other policies? just add some text saying it only applies to 3.1 and later?

Reply:
> just add some text saying it only applies to 3.1 and later?

Yes, I think so.